public/css/nexthome.css

```diff
@@ -1,530 +1,660 @@
+    font-family: "Source Sans Pro", sans-serif;
+    font-size: 16px;
+    font-weight: normal;
+}
+#mainContainer.stats.collaspe .statCardWrap .content .data .count span.subcount {
+    font-size: 16px;
+}
+#mainContainer.stats.collaspe .statCardWrap .content .data .footing {
+    display: none;
+}
+#mainContainer.stats.collaspe .statCardWrap .content .data .progress + .footing {
+    display: block;
+    position: absolute;
+    color: #ffffff;
+    top: 24px;
+    left: calc(5% + 7px);
+    z-index: 99;
+}
 .img-size {
-  height: 20px;
+    height: 20px;
 }
-
 .networkStatsWrap,
 .blockRewardsWrapper,
 .proofofstakeWrapper {
-  padding-right: 50px; }
-.info-block .expanded-block .row .col-12 img {
-    margin-right: 3%;
-    margin-top: 2%;
-    height: 20px; }
+    padding-right: 50px;
+}
+.tippy-tooltip.light-theme {
+    color: #26323d;
+    box-shadow: 0 0 20px 4px rgba(154,161,177,.15), 0 4px 80px -8px rgba(36,40,47,.25), 0 4px 4px -2px rgba(91,94,105,.15);
+    background-color: #fff;
+}
+.tippy-popper[x-placement^=top] .tippy-arrow {
+    border-top-color: white;
+}
+@media (max-width: 1000px) {
+    .netstats-section {
+        width: 100%;
+    }
+    .flex-space {
+        display: none;
+    }
+    .blocks-section {
+        margin-top: 0;
+    }
+}
 @media (min-width: 1400px) {
-  .container {
-    max-width: 1280px !important; }
+    .container {
+        max-width: 1280px !important;
+    }
     .container .logo a img {
-      width:100px !important; } }
+        width: 100px !important;
+    }
+}
 @media (min-width: 800px) {
-  header .right .searchbox.full input {
-    flex: 1; }}
+    header .right .searchbox.full input {
+        flex: 1;
+    }
+}
 @media (max-width: 600px) {
-  header .right .searchbox input {
-    width: 150px; }}
-
-@media only screen and (max-width: 768px)  {
-  .container {
-    max-width: 85% !important;
-  }
-  .container .logo a img {
-    width:70px !important;
-  }
-  .col-md-6 {
-    flex: 0 0 50%;
-    max-width: 100%;
-  }
-  .col-sm-4 {
-    flex: 0 0 33.333333%;
-    max-width: 33.333333%;
-  }
-  .col-lg-4 {
-    flex: 0 0 100%;
-    max-width: 33.333333%;
-  }
-  .networkStatsWrapper .statCard {
-    margin-right: 1% !important;
-    min-height: fit-content;
-  }
-  .dcr-icon {
-    width: 100px;
-  }
-  .dcr-icon-container {
-    width: 100px;
-  }
-}
-
-@media only screen and (max-width: 576px)  {
-  .networkStatsWrapper {
-    padding: 40px 0px;
-  }
-  .decBlockchainWrapper {
-    padding: 0px 0px;
-  }
-  .col-lg-4 {
-    flex: 0 0 100%;
-    max-width: 50%;
-  }
-  .col-sm-4 {
-    flex: 0 0 100%;
-    max-width: 100%;
-    padding-right: 0px;
-    padding-left: 0px;
-  }
-  .dcr-icon {
-    width: 60px;
-  }
-  .dcr-icon-container {
-    width: 60px;
-  }
-  .navbar-fixed-bottom{
-    font-size: 12px;
-  }
+    header .right .searchbox input {
+        width: 150px;
+    }
+}
+@media only screen and (max-width: 768px) {
+    .container {
```

Comment:
On small screens, we should not add extra margin to the side of the content.
This is where screen real estate is at a premium, we should use as much of it as possible for content, not whitespace!

Reply:
That particular style applies to other pages. I had to override that property styling for the /nexthome page using `#nexthome .container` in line 116.
There's still too much space lying fallow on small devices though, caused by a padding of 10% across all screen widths. In line with your comment, I'll reduce the padding in smaller screens to allow the content take up much space.